lilypad/server/api/v0/versions_api.py

```diff
@@ -40,7 +41,7 @@ async def get_version_by_uuid(
 
 
 @versions_router.get(
-    "/projects/{project_uuid}/functions/{function_name}/versions",
+    "/projects/{project_uuid}/functions/name/{function_name}/versions",
```

Comment:
what sparked the inclusion of `/name/`?

Reply:
name conflicts with the other route that got changed. the issue is that we had two routes with the same path but one was using id (int) and the other with (str). that was bad practice which caused a bug when we changed to UUID, which gets converted to str and caused issues

Reply:
got it